Simplified/Reader2/Internal/LibraryService.swift

```diff
@@ -36,14 +36,16 @@ final class LibraryService: NSObject, Loggable {
     #if LCP
     drmLibraryServices.append(LCPLibraryService())
     #endif
+    
+    drmLibraryServices.append(ACSLibraryService())
   }
 
   /// Complementary parsing of the publication.
   /// Will parse Nav/ncx + mo (files that are possibly encrypted)
   /// using the DRM object of the publication.container.
   func loadDRM(for book: NYPLBook, completion: @escaping (CancellableResult<DRM?>) -> Void) {
 
-    guard let filename = book.fileName, let (container, parsingCallback) = items[filename] else {
+    guard let filename = book.fileName, let fileUrl = URL(string: filename), let (container, parsingCallback) = items[fileUrl.lastPathComponent] else {
```

Comment:
We should probably log this. Make sure to log whatever info you can to help us understand what happened.

Reply:
where do you usually log changes like that?

Reply:
it would be a call to a  NYPLErrorLogger method. You can also add a TODO: SIMPLY-2565 for me to finish later.

Reply:
in the code above I fixed an error - LibraryService was trying to get parsed publication by full file name, but the correct key is file name only. `items` dictionary contains [filename: parsingCallback], and probably a better place to log any error is `parsePublication(at url: URL)` function, where the dictionary is being filled with data. I added `TODO: SIMPLY-2656` to the `catch` block there, as in that function we have access to the actual parser error.